src/Serilog/Serilog.csproj

```diff
@@ -60,5 +62,6 @@
     <None Include="..\..\assets\icon.png" Pack="true" Visible="false" PackagePath="" />
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
     <PackageReference Include="Nullable" Version="1.3.1" PrivateAssets="All" />
+    <PackageReference Include="PolySharp" Version="1.12.1" PrivateAssets="All" />
```

Comment:
i think PolySharp makes Nullable redundant?

Reply:
🤔 Maybe. It was not my goal.